package.json

```diff
@@ -12,7 +12,7 @@
     "sonarqube-base-branch": "sonar-scanner",
     "sonarqube-pull-request": "sonar-scanner -D sonar.pullrequest.base=main",
     "start": "node dist/bin/www.js",
-    "dev": "nodemon src/bin/www.ts",
+    "dev": "nodemon --ext ts,njk,json src/bin/www.ts",
```

Comment:
added file extensions to nodemon args to make it reload when changing njk template or json file (e.g. translations.json)

Reply:
and included ts files of course.